assets/cases/middleware/dragonbones/DragonBonesCtrl.ts

```diff
@@ -332,6 +332,21 @@ export default class DragonBonesCtrl extends Component {
         this._bullets.push(bullet);
     }
 
+    simulateTouchStart(x: number, y: number) {
```

Comment:
Should be using the following event simulation in your test case code

```
var event = new EventTouch([], true, InputEventType.TOUCH_START, []);
comp.touchHandler.dispatchEvent(event);
```

Treat the following touch move event and touch end event same as above